assets/js/googlesitekit/datastore/user/date-range.test.js

```diff
@@ -211,5 +227,22 @@ describe( 'core/user date-range', () => {
 				} );
 			} );
 		} );
+
+		describe( 'getReferenceDate', () => {
+			it( 'should return the reference date once set', () => {
+				const someReferenceDate = '2020-08-04';
+
+				registry.dispatch( STORE_NAME ).setReferenceDate( someReferenceDate );
+				expect( registry.select( STORE_NAME )
+					.getReferenceDate() )
```

Comment:
This is a bit of an odd break here 😄 
```suggestion
				expect( registry.select( STORE_NAME ).getReferenceDate() )
```